src/app/organization/offices/offices.component.html

```diff
@@ -60,3 +66,119 @@
   </div>
 
 </div>
+
+<!-- Tree View -->
+
+<div class="container" [hidden]="viewGroup.value !== 'treeView'">
+
+  <div class="m-b-20" fxLayout="row" fxLayoutAlign="start" fxLayoutGap="20px">
+    <button mat-raised-button (click)="nestedTreeControl.expandAll()">  <!-- Bug(angular-material): https://github.com/angular/material2/issues/12170 -->
+      Expand All
+    </button>
+    <button mat-raised-button (click)="nestedTreeControl.collapseAll()">
+      Collapse All
+    </button>
+  </div>
+
+  <div fxLayout="row" fxLayoutGap="4%" fxLayout.lt-md="column">
+
+    <div class="mat-elevation-z8" fxFlex>
```

Comment:
I meant this. The mat-elevation-z8 creates a stronger shadow than the card actually does, and it also does not provide the same edge corners as the card, which I think looks just a tiny bit inconsistent. Again, nothing major, but I just think it might look abit nicer.

Reply:
Thanks for the suggestion, I did not realize that.